### DISCUSSION TITLE
disable image sizes completely -bootcommerce-child-main

### DISCUSSION THREAD

**Discussion by bamhost** [bootscore/bootscore#399] [Q&A]
Hello,

I size and optimize all images used throughout my website myself. I want to completely disable any and all image sizing and optimization that wordpress, and bootcommerce-child-main performs. After I know all image sizing and optimization is disabled, I will then set my own custom sizes specific to my needs in my child theme functions.php file, but first I have to completely disable image sizing everywhere.

I have set image sizes to all 0's in Dashboard > Settings > Media Settings as well as in Appearance > Customize > Woocommerce > Product Images

My question is... does bootscore set image sizes anywhere else...in its core files? If yes, how can I disable those as well?

I realize other plugins may create their own sizes, and I am systematically contacting them as well.

JMO...Wordpress has the worse mismanagement of images I've ever seen, it always has.

Thank you for any help you can give.

**Comment by crftwrk** [comment]
Hi,

bootScore does not set any image sizes. It uses the medium size image in loop `get_the_post_thumbnail(null, 'medium')` .But if there is no medium image, then full size image is used. So, nothing to do here.

> JMO...Wordpress has the worse mismanagement of images I've ever seen, it always has.

Totally agree ;-)
